eo-maven-plugin/src/test/java/org/eolang/maven/RegisterMojoTest.java

```diff
@@ -91,4 +91,37 @@ void doesNotFailWhenNoStrictNames(@TempDir final Path temp) throws IOException {
             Matchers.is(true)
         );
     }
+
+    @Test
+    void registersInExternal(@TempDir final Path temp) throws IOException {
+        new Home(temp).save(
+            new ResourceOf("org/eolang/maven/file-name/abc-def.eo"),
+            Paths.get("src/eo/org/eolang/maven/foo.eo")
+        );
+        final String name = "org.eolang.maven.foo";
+        final String source = "src/eo";
+        final FakeMaven maven = new FakeMaven(temp)
+            .with("sourcesDir", temp.resolve(source).toFile())
+            .execute(new FakeMaven.Register());
+        MatcherAssert.assertThat(
+            String.format(
+                String.join(
```

Comment:
@maxonfjvipon You don't need `String.join` here. Please can you use a single `String.format` here?